src/client/camera.cpp

```diff
@@ -695,16 +695,20 @@ void Camera::drawNametags()
 			i = m_nametags.begin();
 			i != m_nametags.end(); ++i) {
 		Nametag *nametag = *i;
-		v3f cam_pos = m_client->getEnv().getLocalPlayer()->getPosition() +
+		v3f cam_realpos = m_client->getEnv().getLocalPlayer()->getPosition() +
 				m_client->getEnv().getLocalPlayer()->getEyeOffset();
-		v3f nametag_pos = nametag->parent_node->getAbsolutePosition();
+		v3f nametag_realpos = nametag->parent_node->getAbsolutePosition() + intToFloat(m_client->getEnv().getCameraOffset(), BS);
 
+		cam_realpos     /= 10;
+		nametag_realpos /= 10;
```

Comment:
It's probably a better idea to divide by `BS` instead.

Reply:
Yes BS is better, consistent with all other code.